docs/usage/webhook/deploy.rst

```diff
@@ -0,0 +1,64 @@
+.. _usage_ffmpeg_deploy:
+
+Deploying on http server
+========================
+
```

Comment:
Let's make the first section a quick show of the operation, and then move into the dataflow stuff.

```suggestion

.. note::

    All the code for this example is located under the
    `examples/ffmpeg <https://github.com/intel/dffml/blob/master/examples/ffmpeg/>`_
    directory of the DFFML source code.

Writing operations and definitions to convert videos files to gif by calling ``ffmpeg``

**ffmpeg/operations.py**

.. literalinclude:: /../examples/ffmpeg/ffmpeg/operations.py

```